furnace.c

```diff
@@ -550,6 +607,54 @@ do_tcp_work(furnace_context_t *ctx, bool deadline_met)
 
 }
 
+#if CONFIG_SHUTTER
+void
+do_shutter_work(furnace_context_t *ctx)
+{
+  if(ctx->shutter.time_ms != 0){
+    const bool deadline_met = get_absolute_time() > ctx->shutter.deadline;
+    if(deadline_met){
```

Comment:
Please write a comment on the top of `shutter.c` that it was implemented based on the documentation from:
https://holzcoredump.cc/FITEC_FS90.pdf